person.rb

```diff
@@ -22,4 +20,8 @@ def generate_id
   def can_use_services?
     true
   end
+
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] Please make sure the `Person` class is inherited from the `Namebale` class for the associations to work as expected.
  - You can consider the code `class Person < Nameable`